src/dialogs/more-info/more-info-controls.html

```diff
@@ -125,6 +142,11 @@
       };
     }
 
+    enlarge() {
+      this.large = !this.large;
```

Comment:
Should we use `setProperties` here?

Reply:
Removed cache config change - it didn't  do anything.